modules/yieldlabBidAdapter.js

```diff
@@ -198,7 +201,13 @@ function createQueryString (obj) {
   let str = []
   for (var p in obj) {
     if (obj.hasOwnProperty(p)) {
-      str.push(encodeURIComponent(p) + '=' + encodeURIComponent(obj[p]))
+      let param = p
```

Comment:
Why is variable `param` needed? Can `p` be used in those places, too?

Reply:
Can be used, yes. Changed.